qt/python/mantidqt/py3compat.py

```diff
@@ -0,0 +1,71 @@
+#  This file is part of the mantid workbench.
+#
+#  Copyright (C) 2017 mantidproject
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+mantidqt.py3compat
+------------------
+
+Transitional module providing compatibility functions intended to help
+migrating from Python 2 to Python 3. Mostly just wraps six but allowing
+for additional functionality of our own.
+
+This module should be fully compatible with:
+    * Python >=v2.7
+    * Python 3
+"""
+
+from __future__ import absolute_import, print_function
+
+import six
+from six import * # noqa
+
+# -----------------------------------------------------------------------------
+# Globals and constants
+# -----------------------------------------------------------------------------
+__all__ = dir(six)
+
+
+# -----------------------------------------------------------------------------
+# Strings
+# -----------------------------------------------------------------------------
+def is_text_string(obj):
+    """Return True if `obj` is a text string, False if it is anything else,
+    like binary data (Python 3) or QString (Python 2, PyQt API #1)"""
+    if PY2:
+        # Python 2
+        return isinstance(obj, basestring)
```

Comment:
Why not use `six.string_types`? Then you don't need the `if`, simply `return isinstance(obj, string_types)`

Reply:
:+1: